quodlibet/po/fr.po

```diff
@@ -2389,186 +2381,189 @@ msgstr "Theme natif"
+#: ../quodlibet/qltk/songsmenu.py:340
 msgid "Play_lists"
-msgstr "Listes de lecture"
+msgstr "_Listes de lecture"
 
-#: ../quodlibet/ext/events/trayicon/menu.py:134
+#: ../quodlibet/ext/events/trayicon/menu.py:132
 #: ../quodlibet/qltk/debugwindow.py:195
-#: ../quodlibet/qltk/quodlibetwindow.py:1024
+#: ../quodlibet/qltk/quodlibetwindow.py:1025
 msgid "_Quit"
 msgstr "_Quitter"
 
-#: ../quodlibet/ext/events/trayicon/prefs.py:41
+#: ../quodlibet/ext/events/trayicon/prefs.py:42
 #, fuzzy
 msgid "Scroll wheel adjusts volume"
 msgstr "La mollette de la souris change le volume"
 
-#: ../quodlibet/ext/events/trayicon/prefs.py:47
+#: ../quodlibet/ext/events/trayicon/prefs.py:48
 #, fuzzy
 msgid "Scroll wheel changes song"
 msgstr "La molette de la souris change la musique"
 
-#: ../quodlibet/ext/events/trayicon/prefs.py:53
+#: ../quodlibet/ext/events/trayicon/prefs.py:54
 msgid "Scroll _Wheel"
 msgstr ""
 
-#: ../quodlibet/ext/events/trayicon/prefs.py:65
-#: ../quodlibet/ext/events/trayicon/prefs.py:86
-msgid "tooltip"
-msgstr ""
-
-#: ../quodlibet/ext/events/trayicon/prefs.py:81
-#, fuzzy
+#: ../quodlibet/ext/events/trayicon/prefs.py:82
 msgid "Tooltip Display"
-msgstr "Éditer l'affiche des informations"
+msgstr "Suggestions Affichage"
 
 #: ../quodlibet/ext/events/viewlyrics.py:23
-#, fuzzy
 msgid "View Lyrics"
-msgstr "Paroles"
+msgstr "Voir les paroles"
 
 #: ../quodlibet/ext/events/viewlyrics.py:24
 msgid "Automatically displays lyrics beneath the song list in the main window."
 msgstr ""
 
 #: ../quodlibet/ext/events/viewlyrics.py:29
-#, fuzzy
 msgid "_Lyrics"
-msgstr "Paroles"
+msgstr "Paro_les"
+
+#: ../quodlibet/ext/events/visualisations.py:36
+msgid "Launch Visualisations"
+msgstr ""
+
+#: ../quodlibet/ext/events/visualisations.py:38
+msgid "Launch external visualisations."
+msgstr ""
+
+#: ../quodlibet/ext/events/visualisations.py:55
+#, python-format
+msgid "Couldn't run visualisations using '%s'"
+msgstr ""
+
+#: ../quodlibet/ext/events/visualisations.py:57
+#: ../quodlibet/ext/songsmenu/fingerprint/search.py:44
+msgid "Error"
+msgstr "Erreur"
+
+#: ../quodlibet/ext/events/visualisations.py:76
+msgid "Visualiser executable:"
+msgstr ""
 
-#: ../quodlibet/ext/events/waveformseekbar.py:292
+#: ../quodlibet/ext/events/visualisations.py:91
+msgid "Reload"
+msgstr "Recharger"
+
+#: ../quodlibet/ext/events/waveformseekbar.py:297
 msgid "Waveform Seek Bar"
 msgstr ""
 
-#: ../quodlibet/ext/events/waveformseekbar.py:296
+#: ../quodlibet/ext/events/waveformseekbar.py:301
 msgid "A seekbar in the shape of the waveform of the current song."
 msgstr ""
 
-#: ../quodlibet/ext/events/waveformseekbar.py:329
+#: ../quodlibet/ext/events/waveformseekbar.py:334
 msgid "Override foreground color:"
 msgstr ""
 
-#: ../quodlibet/ext/events/waveformseekbar.py:340
+#: ../quodlibet/ext/events/waveformseekbar.py:345
 msgid "High Res"
 msgstr ""
 
 #: ../quodlibet/ext/events/write_cover.py:32
 msgid "Picture Saver"
-msgstr ""
+msgstr "Économiseur d'écran"
```

Comment:
That plugin "saves the cover image of the current song to a file" so its name is literally "Enregistreur d'image", so something like "Enregistrement de la jaquette" or "Sauvegarde de jaquette"